templates/worker-turso-ts/src/index.ts

```diff
@@ -0,0 +1,82 @@
+import { Client as LibsqlClient, createClient } from '@libsql/client/web';
+import { Router, RouterType } from 'itty-router';
+
+export interface Env {
+    // The environment variable containing your the URL for your Turso database.
+    LIBSQL_DB_URL?: string;
+    // The Secret that contains the authentication token for your Turso database.
+    LIBSQL_DB_AUTH_TOKEN?: string;
+
+    // These objects are created before first use, then stashed here
+	// for future use
+    client?: LibsqlClient;
+    router?: RouterType;
+}
+
+export default {
+    async fetch(request: Request, env: Env): Promise<Response> {
+        if (env.router === undefined) {
+            env.client = buildLibsqlClient(env);
+            env.router = buildRouter(env);
+        }
+
+        return env.router.handle(request);
+    },
+};
+
+function buildLibsqlClient(env: Env): LibsqlClient {
+    const url = env.LIBSQL_DB_URL;
+    if (url === undefined) {
+        throw new Error('LIBSQL_DB_URL env var is not defined')
+    }
+
+    const authToken = env.LIBSQL_DB_AUTH_TOKEN;
+    if (authToken === undefined) {
+        throw new Error('LIBSQL_DB_AUTH_TOKEN env var is not defined')
+    }
+
+    return createClient({url, authToken});
+}
```

Comment:
The example in https://github.com/cloudflare/cloudflare-docs/pull/8302 calls `trim()` on the `url` and `authToken` vars so I'm not too sure which one is more recent - but if this example intends to call `trim()` then we'll want to update both to use `?.trim()` to avoid throwing when the variable is undefined.